Automattic/pocket-casts-ios
# Fix crash audio high filter

Fixes [this crash](https://a8c.sentry.io/issues/4238066024/?project=4503921846583296&query=is:unresolved+issue.priority:%5Bhigh,+medium%5D+release.build:%227.63*%22&statsPeriod=7d&stream_index=0)

This PR replaces the use of `unsafeBitCast` to use Unmanaged as recommended in the documentation [here](https://developer.apple.com/documentation/swift/unsafebitcast(_:to:))


## To test

- Start the app
- Start playing a podcast with Volume Boost enabled and Trim Silence Disabled
- Play around forward, backward, skipping forward and stopping on the player to see if all works correctly

## Checklist
